### DISCUSSION TITLE
Adding manifest.json and assetlinks.json file in ScandiPWA

### DISCUSSION THREAD

**Discussion by veppywaqar** [scandipwa/scandipwa#5577] [Q&A]
Hello Everyone

i am generating APK for Our site using bubblewrap, i created a apk sucessfully but to hide url bar in chrome i need to add .wellknow/assetlink.json file

Anyone please share your experience anyone knows

**Comment by veppywaqar** [comment]
Just for an Update

we have added  assetslink file in pub/.wellknown folder and it worked fine

**Reply by Ylmzef** [reply]
Did you use pwabuilder.com for this?
